apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( 'a8c/navigation-menu', {
+	title: __( 'Navigation Menu' ),
+	description: __( 'Visual placeholder for site-wide navigation and menus.' ),
+	icon: 'layout',
```

Comment:
@jasmussen do you happen to have the SVG for the icon in the related mock? 

This is non-blocking @kwight, we can always follow up on this.

Reply:
Sure, it's this one: https://material.io/tools/icons/static/icons/outline-navigation-24px.svg

Reply:
Thanks!